src/platforms/stm32/src/lib/gpio_driver.c

```diff
@@ -899,22 +845,18 @@ static NativeHandlerResult consume_gpio_mailbox(Context *ctx)
             break;
 
         case GPIOInvalidCmd:
-            char *invalid_name = interop_atom_to_string(ctx, cmd_term);
-            AVM_LOGE(TAG, "Invalid command: %s", invalid_name);
-            free(invalid_name);
-            ret = create_pair(ctx, ERROR_ATOM, UNDEFINED_ATOM);
+            AVM_LOGE(TAG, "Invalid command");
+            ret = UNDEFINED_ATOM;
```

Comment:
Also maybe `FUNCTION_CLAUSE_ATOM` here, too?

Reply:
This makes sense here for sure.